modules/tensor_mechanics/src/materials/ComputeMeanThermalExpansionEigenstrainBase.C

```diff
@@ -26,17 +26,6 @@ ComputeMeanThermalExpansionEigenstrainBase::ComputeMeanThermalExpansionEigenstra
 }
 
 void
-ComputeMeanThermalExpansionEigenstrainBase::initialSetup()
-{
-  _alphabar_stress_free_temperature = meanThermalExpansionCoefficient(_stress_free_temperature);
-  _thexp_stress_free_temperature =
-      _alphabar_stress_free_temperature * (_stress_free_temperature - referenceTemperature());
```

Comment:
I should probably take the time to understand what I'm doing before I do it.  Anyway, made the fix.